FlowCrypt/Controllers/Inbox/InboxProviders.swift

```diff
@@ -29,7 +29,9 @@ class InboxMessageThreadsProvider: InboxDataProvider {
 
     override func fetchInboxItems(using context: FetchMessageContext) async throws -> InboxContext {
         let result = try await provider.fetchThreads(using: context)
-        let inboxData = result.threads.map(InboxRenderable.init)
+        let inboxData = result.threads.map { thread in
+            return InboxRenderable(thread: thread, folderPath: context.folderPath ?? "")
+        }
```

Comment:
Not sure what is an `` (empty string) path. Doesn't lead anywhere - it's as bad as a nil. 

Any good reason why `context.folderPath` should ever be nil? I mean, why is `folderPath` nilable in the first place?

Reply:
I don't know TBH, it was done a long time before I entered project.

In scope of this task, even if its really sometimes nil, it wont happen in SENT folder, and in this case our code will work totally fine, because it is only purpose of passing folder there.

also, from my brief review, this optional has more usage in our app via `MessageSearchContext`.
if you want me to explore this and remove optionals from `FetchMessageContext` we can do that in scope of another task, because its out of current task scope.

Reply:
filed here https://github.com/FlowCrypt/flowcrypt-ios/issues/1101